gamemodes/terrortown/gamemode/shared/events/base_event.lua

```diff
@@ -15,6 +16,7 @@ end
 EVENT.type = "base_event"
 EVENT.event = {}
 EVENT.score = {}
+EVENT.karma = {}
 EVENT.plys = {}
 EVENT.plys64 = {}
```

Comment:
While those are saved as plys64 but that is used all around your event system, so it would be a major change

Reply:
I don't understand why this would be a major change. I just have to rename the variable, that's it.

Reply:
It's used everywhere. My main point is anyway, that this shouldn't be done with this PR here. Just wanted to point it out.